app/knights.py

```diff
@@ -0,0 +1,38 @@
+class Knights:
+    # This constructor is creating a knight (Knights class instance),
+    # which is ready for the battle
+    # (armoured and potion applied (if available)).

```

Comment:
Simplify the description and make it a docstring instead of a comment

Reply:
By simplify i mean don't add that knight is Knights class instance, it is obvious. So you don't need it in class description